extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -346,7 +355,7 @@ private static File prepareDestinationFileLocation(@NotNull File destination) {
     }
 
     /**
-     * Helper function for building grouping column info for writing and deleting any backup grouping column files
+     * Helper function for building grouping column info for writing. Also, deletes any backup grouping column files.
```

Comment:
Does this delete `any` backup files or just similarly named files?

```suggestion
     * Helper function for building grouping column info for writing. Also deletes any conflicting backup grouping column files.
```

Reply:
Just similarly named files.